openstack/cbr/v3/tags/results.go

```diff
@@ -0,0 +1,96 @@
+package tags
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/cbr/v3/vaults"
+)
+
+type Tag struct {
+	Key    string   `json:"key"`
+	Values []string `json:"values"`
+}
+
+type TagResult struct {
+	golangsdk.Result
+}
+
+func (r TagResult) Extract() ([]Tag, error) {
+	var s struct {
+		Tags []Tag `json:"tags"`
+	}
+	if r.Err != nil {
+		return nil, r.Err
+	}
+	err := r.ExtractInto(&s)
+	if err != nil {
+		return nil, fmt.Errorf("failed to extract Vault Project Tags")
+	}
+	return s.Tags, nil
+}
+
+// ----------------------------------------------------------------------------
+
+type SysTag struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type InstancesResponse struct {
+	Resources  []TagResource `json:"resources"`
+	TotalCount int           `json:"total_count"`
+}
+
+type TagResource struct {
+	ResourceID     string   `json:"resource_id"`
+	ResourceDetail []Vault  `json:"resource_detail"`
+	Tags           []SysTag `json:"tags"`
+	ResourceName   string   `json:"resource_name"`
+	SysTags        []SysTag `json:"sys_tags"`
+}
+
+type Vault struct {
+	vaults.Vault
+	SmnNotify bool `json:"smn_notify"`
+	Threshold int  `json:"threshold"`
+}
```

Comment:
strange structure, do you need it here?

Reply:
We do not support these two fields and have removed them.